index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+ <div class="nav">
+    <navbar id="navbar">
+      <a href="#">About</a>
+      <a href="#">Projects</a>
+      <a href="#">Contacts</a>
+    </navbar>
+    </div>
+    <div id="welcome-section">
```

Comment:
The same applies here, i think it would be better if you use `section` tag in place of `div` tag.

`section` tag is an example of semantic tags.